src/hub_predtimechart/app/generate_target_json_files.py

```diff
@@ -53,8 +55,41 @@ def main(hub_dir, target_out_dir):
 
 
 def get_target_data_df(hub_dir, target_data_filename):
-    # load the target data csv file from the hub repo for now, file path for target data is hard coded
-    return pl.read_csv(hub_dir / 'target-data' / target_data_filename, null_values=["NA"])
+    """
+    Loads the target data csv file from the hub repo for now, file path for target data is hard coded to 'target-data'.
+    Raises FileNotFoundError if target data file does not exist.
+    """
+    schema = {
+        "location": pl.String,  # to handle the 'US' location (the only one that doesn't parse as Int64)
+        # "date": pl.Date,
+        # "value": pl.Int64
+    }
+    target_data_file_path = hub_dir / 'target-data' / target_data_filename
+    try:
+        return pl.read_csv(target_data_file_path, schema_overrides=schema, null_values=["NA"])
+    except FileNotFoundError as error:
+        raise FileNotFoundError(f"target data file not found. {target_data_file_path=}, {error=}")
+
+
+def ptc_target_data(target_data_df: pl.DataFrame, task_ids_tuple: tuple[str]):
+    """
+    Returns a dict for a single reference date in the target data format documented at https://github.com/reichlab/predtimechart?tab=readme-ov-file#fetchdata-truth-data-format.
+    That is, looking at that example, this function returns the date and value columns as in
+    tests/expected/FluSight-forecast-hub/target/wk-inc-flu-hosp_US.json :
+
+    {
+      "date": ["2024-04-27", "2024-04-20", "..."],
+      "y": [2337, 2860, "..."]
+    }
```

Comment:
Suggestions to update docs to clarify assumptions/limitations of the current code:
```suggestion
    Returns a dict for a single reference date and location in the target data format documented at https://github.com/reichlab/predtimechart?tab=readme-ov-file#fetchdata-truth-data-format.
    Note that this function currently assumes there is only one task id variable other than the
    reference date, horizon, and target date, and that task id variable is a location code that
    matches codes used in the `location` column of the `target_data_df` argument.
    That is, looking at that example, this function returns the date and value columns as in
    tests/expected/FluSight-forecast-hub/target/wk-inc-flu-hosp_US.json :

    {
      "date": ["2024-04-27", "2024-04-20", "..."],
      "y": [2337, 2860, "..."]
    }
```